regolith/helpers/a_projectumhelper.py

```diff
@@ -122,7 +127,7 @@ def db_updater(self):
             'name': rc.name,
             'pi_id': rc.pi_id,
             'lead': rc.lead,
-            'notes': rc.notes,
+            'notes': rc.notes + "\nGoogle Doc: " + docs_address + "\n GitLab Project: ",
```

Comment:
don't forget to use f-strings instead of summing strings.

Also, I think there specific url fields for htis information that iwll make them easier to find later, than putting them in notes, though this was a good idea.  If not, we should crete them.  Have a look in schema.py to see.